src/index.js

```diff
@@ -621,22 +622,23 @@ class Manager {
       return this._spawn(binPath, sanityCheck.args);
     })
     .then((output) => {
-      const haystack = output.stdout + output.stderr;
+      this._logger.debug(`Sanity check output: ${output.stdout + output.stderr}`);
 
-      this._logger.debug(`Sanity check output: ${haystack}`);
-
-      const needles = sanityCheck.output || [];
+      try {
+        var version = output.stdout.match(/\d+\.\d+\.\d+/)[0];
```

Comment:
We should not be hardcoding version checks. The `sanityCheck.output` configuration key is meant to contain the output that needs to checked for.